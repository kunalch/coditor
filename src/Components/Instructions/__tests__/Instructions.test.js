import React from "react";
import renderer from "react-test-renderer";
import Instructions from "./../Instructions";

describe("<Instructions /> ", () => {
  it("should render properly", () => {
    const tree = renderer.create(
      <Instructions />
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
