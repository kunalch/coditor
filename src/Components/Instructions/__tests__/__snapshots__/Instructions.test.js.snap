// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Instructions />  should render properly 1`] = `
<ul
  className="text-base Instructions"
>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
  <li
    className="pb-3"
  >
    <i
      class="icon-check-circle mr-2"
    />
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  </li>
</ul>
`;
